tests/test_superform.py

```diff
@@ -64,4 +80,43 @@ def test_validation(self):
         self.assertEqual(form.is_valid(), False)
         self.assertTrue(form.errors['username'])
         self.assertTrue(form.errors['emails'])
-        self.assertIsInstance(form.errors['emails'], ErrorList)
+        self.assertIsInstance(form.errors['emails'], list)
```

Comment:
You're completely correct, however this is actually the way Django formsets currently work.

```
>>> from django.conf import settings
>>> settings.configure()
>>> from django.core.wsgi import get_wsgi_application
>>> _ = get_wsgi_application()
>>> from tests.test_superform import EmailFormSet
>>> fset = EmailFormSet({'form-TOTAL_FORMS': '1', 'form-INITIAL_FORMS': '0', 'form-MIN_NUM_FORMS': '0', 'form-0-email': 'foo'})
>>> fset.errors
[{'email': [u'Enter a valid email address.']}]
>>> type(fset.errors)
<type 'list'>
>>> type(fset.errors[0])
<class 'django.forms.utils.ErrorDict'>
```

Although now that I think about it I'm not entirely sure how I wound up in a situation where I specifically wanted a `list` instead of an `ErrorList`. I'll have to try out the code without it again; I'm sure I can get it to work with an `ErrorList`.